integration/obscurogateway/obscurogateway_test.go

```diff
@@ -317,3 +434,69 @@ func waitServerIsReady(serverAddr string) error {
 	}
 	return fmt.Errorf("timed out before server was ready")
 }
+
+func TransferETHToAddress(client *ethclient.Client, wallet wallet.Wallet, toAddress gethcommon.Address, amount int64) (*types.Receipt, error) {
+	transferTx1 := types.LegacyTx{
+		Nonce:    wallet.GetNonceAndIncrement(),
+		To:       &toAddress,
+		Value:    big.NewInt(amount),
+		Gas:      uint64(1_000_000),
+		GasPrice: gethcommon.Big1,
+		Data:     nil,
+	}
+	signedTx, err := wallet.SignTransaction(&transferTx1)
+	if err != nil {
+		return nil, err
+	}
+	err = client.SendTransaction(context.Background(), signedTx)
+	if err != nil {
+		return nil, err
+	}
+	return integrationCommon.AwaitReceiptEth(context.Background(), client, signedTx.Hash(), 2*time.Second)
+}
+
+func getStringValueFromSmartContractGetter(contractAddress gethcommon.Address, contractAbi abi.ABI, method string, client *ethclient.Client) (string, error) {
```

Comment:
Perhaps this wraping func can be removed and used the 2 funcs like in the errors test.

Reply:
removed